README.md

```diff
@@ -13,35 +13,35 @@ Investigator study management and progress tracking.
 
 ## Development Quick Start
 
+If you haven't already, checkout the kf-ui-data-tracker master branch:
+
+```
+git clone https://github.com/kids-first/kf-ui-data-tracker.git
+cd kf-ui-data-tracker
+```
+
 #### Setup backend API service
 
 Study-creator served as backend provides the data APIs consumed by data-tracker frontend.
 
-If you haven't already, create the kf-data-stack network:
+If you haven't already, create the kf-data-stack network and necessary external volume:
 
 ```
 docker network create kf-data-stack
+docker volume create --name=bundle
 ```
 
-Checkout kf-api-study-creator master and bring up the study creator API:
-Checkout the [Kids First Data Stack](https://github.com/kids-first/kf-data-stack)
-and make sure the study creator services are running:
+Next bring up the study creator API in development mode:
```

Comment:
Not really sure what was supposed to go here. Do we use kf-data-stack docker-compose, study-creator docker-compose or data-tracker docker-compose?

Reply:
The docker-compose included in this repo sets up the study creator and all other apis needed for testing. The line you have should work for just bringing up the essential study creator api.